src/devices/video/mc6845.cpp

```diff
@@ -473,23 +493,27 @@ void mc6845_device::recompute_parameters(bool postload)
 			// Also, the mode-register change needs to be added to the changed-parameter tests above.
 			if (MODE_INTERLACE_AND_VIDEO)
 			{
-				//max_visible_y *= 2;
+				//visible_height *= 2;
 				//vert_pix_total *= 2;
 			}
 
 			if(m_show_border_area)
 				visarea.set(0, horiz_pix_total-2, 0, vert_pix_total-2);
 			else
-				visarea.set(0 + m_visarea_adjust_min_x, max_visible_x + m_visarea_adjust_max_x, 0 + m_visarea_adjust_min_y, max_visible_y + m_visarea_adjust_max_y);
+				visarea.set(0 + m_visarea_adjust_min_x, visible_width + m_visarea_adjust_max_x, 0 + m_visarea_adjust_min_y, visible_height + m_visarea_adjust_max_y);
 
-			LOGCONF("M6845 config screen: HTOTAL: %d  VTOTAL: %d  MAX_X: %d  MAX_Y: %d  HSYNC: %d-%d  VSYNC: %d-%d  Freq: %ffps\n",
-				 horiz_pix_total, vert_pix_total, max_visible_x, max_visible_y, hsync_on_pos, hsync_off_pos - 1, vsync_on_pos, vsync_off_pos - 1, refresh.as_hz());
+			LOGCONF("M6845 config screen: HTOTAL: %d  VTOTAL: %d  VIS_WIDTH: %d  VIS_HEIGHT: %d  HSYNC: %d-%d  VSYNC: %d-%d  Freq: %ffps\n",
+				 horiz_pix_total, vert_pix_total, visible_width, visible_height, hsync_on_pos, hsync_off_pos - 1, vsync_on_pos, vsync_off_pos - 1, refresh.as_hz());
 
 			if (has_screen())
-				screen().configure(horiz_pix_total, vert_pix_total, visarea, refresh.as_attoseconds());
+			{
+				screen().configure(horiz_pix_total - visual_adjustment, vert_pix_total - visual_adjustment, visarea, refresh.as_attoseconds());
+			}
 
 			if(!m_reconfigure_cb.isnull())
-				m_reconfigure_cb(horiz_pix_total, vert_pix_total, visarea, refresh.as_attoseconds());
+			{
+				m_reconfigure_cb(horiz_pix_total - visual_adjustment, vert_pix_total - visual_adjustment, visarea, refresh.as_attoseconds());
+			}
```

Comment:
You don’t inset the total width/height – those are counts, not inclusive coordinates.